src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.html

```diff
@@ -261,7 +261,7 @@ <h1>Enrollment Results for {{ courseid }}</h1>
   </div>
   <div *ngFor="let enrollResultPanel of enrollResultPanelList">
     <div class="card enroll-results-panel" *ngIf="enrollResultPanel.studentList.length > 0">
-      <div class="card-header {{ enrollResultPanel.panelClass }}">
+      <div class="card-header {{ panelClass[enrollResultPanel.status] }}">
```

Comment:
Try ngClass https://angular.io/api/common/NgClass.
```suggestion
      <div class="card-header", [ngClass]="{'bg-primary': enrollResultPanel.status === EnrollStatus.NEW, 'bg-warning': enrollResultPanel.status === EnrollStatus.MODIFIED}">
```